modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/util/OperationUtils.java

```diff
@@ -213,7 +211,7 @@ public void dataWipe(Status status, SyncmlDocument syncmlDocument,
             throws OperationManagementException, DeviceManagementException {
 
         if (status.getData().equals(Constants.SyncMLResponseCodes.ACCEPTED)) {
-            pendingDataOperations = SyncmlUtils.getDeviceManagementService()
+            pendingDataOperations = WindowsAPIUtils.getDeviceManagementService()
```

Comment:
this is not a util class,therefore refactore it from https://github.com/wso2/product-mdm/pull/156